arc/database/api.py

```
@@ -378,6 +408,108 @@ async def record_batch_outcomes(
+            if isinstance(scenario, dict):
+                scenario_id = scenario.get("id", f"scenario_{call_index}")
+            elif hasattr(scenario, 'id'):
+                scenario_id = scenario.id
+            else:
+                scenario_id = f"scenario_{call_index}"
+            
+            # Transform Modal result to outcome format
+            outcome_data = {
+                "simulation_id": simulation_id,
+                "scenario_id": scenario_id,
+                "execution_time": datetime.now(timezone.utc),
```

Ruff (0.11.9)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)